.github/stability.md

```diff
@@ -0,0 +1,59 @@
+## Stability
+
+In short: `windows-rs` is still under active development, and none of the crates
+under the org are considered "stable". Some crates such as `windows-sys` might
+be closer than others, but until a crate reaches `1.0.0` it may be subject to
+change.
+
+## Release Schedule and Versioning
+
+We're set to release new versions of crates every 3 weeks on Tuesdays. Currently all crates
+are pre-1.0 and have their semver-minor version bumped on release. We are
+looking for ways to version and release crates individually. Once we do, we will
+update this section with our versioning strategy.
+
+## Metadata Stability
+
+All crates in `windows-rs` interact with [Windows Metadata (`.winmd`)
+files](https://github.com/microsoft/win32metadata) which describe the various
+Windows APIs. All Windows APIs can be split into two categories:
+`win32/COM` APIs and `WinRT` APIs. The way we bind to these APIs is different,
+but in the metadata they also have different stability guarantees:
+
+| Metadata category | Stable? |
+|-------------------|---------|
+| `Win32/COM`       | ❌       |
+| `WinRT`           | ✅       |
+
+When metadata is marked as "stable" it means that it will only ever be extended.
+Information will not be changed or removed.
```

Comment:
With this understanding of _stable_ (thanks for defining it!), I don't think we can say the WinRT bucket is stable.

WinRT APIs get added or get marked deprecated regularly, typically during OS release windows. This results in changes to the metadata this crate relies upon that could then critically alter the crate feature map, causing breakage.

---

A somewhat contrived but real world example:

Rewind your clocks to Windows 8.1 timeframe. Let's envision this shiny new API shipped: `GattDeviceService.GetIncludedServices`

It's currently described in Windows Metadata as:
```
[MethodImpl(MethodImplOptions.InternalCall)]
public extern IReadOnlyList<GattDeviceService> GetIncludedServices([In] Guid serviceUuid);
```

Looks good, I slam `features = ["Windows_Devices_Bluetooth"]`" into my crate and life is good.

Some time passes, Microsoft sweeps Windows 8 under the carpet and puts out Windows 10. This Windows 8-era API is subsequently deprecated.

It is now described in Windows Metadata as:

```
[MethodImpl(MethodImplOptions.InternalCall)]
[Deprecated("Use GetIncludedServicesAsync instead of GetAllIncludedServices.  For more information, see MSDN.", DeprecationType.Deprecate, 262144u, "Windows.Foundation.UniversalApiContract")]
public extern IReadOnlyList<GattDeviceService> GetAllIncludedServices();
```

The `windows` crate then picks up newer metadata at some point and pushes out a point release. Now because this API is lumped into the `deprecated` feature bucket, my crate breaks.

---

I think your stability goal is reachable. But we're going to need to sit down and answer a few more questions before we can begin to really make any promises. (Forgive me if you already have these answers, add them to doc. if so!)

Some questions:

* Which versions of Windows does the crate target? How do APIs deprecated mid-OS-major-release-cycle get handled? (An increasing number of Windows components are shipping/updating out of band...)
* Do we have a clear understanding of the upstream sources and their release schedules? Any stability guarantees there? win32metadata, to my (limited external) knowledge, has no non-prerelease release in sight. And OS metadata is a toss up (see earlier example). Are we going to need to slice and dice up upstream metadata?
* What's the expected service level agreement (SLA)? What's my recourse if Microsoft breaks its promise? Will it fix the crate immediately? Drag its feet until next release? Can I open a ticket with Product Support and pay money? What if Kenny is out for vacation?